packages/strapi-parts/src/Link/Link.stories.mdx

```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
 
 # Link
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
+- Clicking on a Link opens a new tab.
+- Links should always be used with an icon. If it is an external link, use the ExternalLink icon accordingly.
+- Links text should describe an action and start with a verb.
+- Links text should be concise and describe their purpose. Users need to know exactly what they are about to do.
+- Use capital letters.
```

Comment:
Nope, if we want capital letters, we'll do it on the styling side, not in the code

Reply:
In the new admin views, sometimes, there is no capital letters 

![Screenshot 2021-08-17 at 09 55 12](https://user-images.githubusercontent.com/7756284/129686693-7d4d7c3d-e1d1-4755-94ca-ab7783e3722f.png)

Reply:
Hicham: The idea is to have consistency, so if it's not capital letter here, this is a mistake coming from the design side 😅 
Marvin: We don't do it yet on the styling side?

Reply:
What I mean is that if we want to transform the text to be in uppercase, it won't be the job of the developer to type things in uppercase. Imagine a blind user navigating and be yelled on the face saying "LOG IN VIA STRAPI YOU COWARD" 😛 .

When done on the design side (I mean, on the CSS part of the thing, automatically, without a developer action), the text will be announced less agressivel :).

TL;DR => people just write "Log in via Strapi" and we transform the text for them on the according component

Reply:
@mfrachet If you look at the UI documentation, you'll see that the Link component is supposed to have a Capital letter style. I think Hicham is not talking about the button, but the "Forgot your password" below. I mentioned that if it's not capital letter, then it's a design mistake :) (if the semantic is fine, which is think so? 🤔 )